src/curl_async.c

```diff
@@ -194,7 +191,7 @@ void *swupd_curl_parallel_download_start(size_t max_xfer)
 
 	h->max_xfer = max_xfer;
 	h->curl_hashmap = hashmap_new(SWUPD_CURL_HASH_BUCKETS, file_hash_cmp, file_hash_value);
-	h->timeout = RETRY_TIMEOUT;
+	h->timeout = retry_delay;
```

Comment:
Can you rename h->timeout too? Because it's not a timeout

Reply:
Good idea

Reply:
Done